app/src/main/java/ar/com/wolox/android/training/ui/training/main/tabs/news/NewsAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package ar.com.wolox.android.training.ui.training.main.tabs.news
+
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.training.model.NewsItem
+import com.facebook.drawee.view.SimpleDraweeView
+import org.ocpsoft.prettytime.PrettyTime
+import java.util.Locale
+
+class NewsAdapter(private val dataSet: List<NewsItem>, private val clickListener: (NewsItem) -> Unit) : RecyclerView.Adapter<NewsAdapter.NewsViewHolder>() {
+
+    class NewsViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
+            RecyclerView.ViewHolder(inflater.inflate(R.layout.news_item, parent, false)) {
+        private var mUserView: TextView? = null
```

Comment:
This will be deleted and not a big deal, but remember the `m...` is a Java convention.